addons/udes_stock/models/stock_move_line.py

```diff
@@ -315,10 +315,11 @@ def prepare(
         Example: [{"barcode": "PROD01", "uom_qty":1}]
 
         """
+        Package = self.env["stock.quant.package"]
         Product = self.env["product.product"]
 
         vals = {}
-        if result_package:
+        if isinstance(result_package, type(Package)):
```

Comment:
q: what else can `result_package` be?

Reply:
`None`

Reply:
Maybe a small comment to explain why we doing this: be able to remove (set false) that field

Reply:
Similar to the commit message :)